cmd/root.go

```diff
@@ -72,6 +81,7 @@ var (
 				rootCmdOpts.watchAvailableStorageInterval,
 				rootCmdOpts.watchAvailableStorageMinBytes,
 				rootCmdOpts.lowAvailableStorageAction,
+				acPolicyConfig,
```

Comment:
any reason to create this struct earlier? we can also keep going with flat arguments here, there's not much structure anyway.

```suggestion
			generic.AdmissionControlPolicyConfig{
				PolicyName:         rootCmdOpts.admissionControlPolicy,
				LimitMaxRunningTxn: rootCmdOpts.acpLimitMaxRunningTxn,
			}
```

Reply:
No strong reason. Let's keep the parameter list flat.